### PR TITLE
Mobile REST API Template

### PR DESCRIPTION
**Pull request by malinajirka**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This PR introduces a new GitHub issue template for the mobile teams, ensuring each API change request contains all necessary information and is consistent. More context on p91TBi-cob#comment-13409-p2.

### How to test the changes in this Pull Request:

[Check how the template looks like](https://github.com/woocommerce/woocommerce/blob/rest-api-mobile-template/.github/ISSUE_TEMPLATE/rest-api-mobile.yml) - any feedback is welcome as always!

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [x] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [x] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

This change doesn't have any user or developer facing impact and therefore doesn't require a changelog entry.

</details>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/rest-api-mobile.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: ✨ REST API MOBILE
```
Comment (coreymckrill) [inline .github/ISSUE_TEMPLATE/rest-api-mobile.yml]:
```suggestion
name: 📱 REST API issue for mobile apps
```

Let's change the emoji to differentiate it a bit from the regular Enhancement request. Also making the title a bit more explicit. What do you think?

File: .github/ISSUE_TEMPLATE/rest-api-mobile.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+description: If you need to modify an existing REST API endpoint or introduce a new one, please let us know or submit a Pull Request!
```
Comment (coreymckrill) [inline .github/ISSUE_TEMPLATE/rest-api-mobile.yml]:
```suggestion
description: Bug reports and enhancement requests for the REST API related to its usage by the native mobile apps.
```

File: .github/ISSUE_TEMPLATE/rest-api-mobile.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+labels: ["type: task", "focus: rest api", "stakeholder: mobile teams"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Thanks for contributing!
+
+        Please provide the information requested in this form.
+        
+        P.S. If you want to request changes to the native mobile apps, please create your request directly in the [iOS](https://github.com/woocommerce/woocommerce-ios/issues) or [Android](https://github.com/woocommerce/woocommerce-android/issues) repositories.
```
Comment (coreymckrill) [inline .github/ISSUE_TEMPLATE/rest-api-mobile.yml]:
```suggestion
        **Note:** To report a bug or request an enhancement to the native mobile apps, please create your request directly in the [iOS](https://github.com/woocommerce/woocommerce-ios/issues) or [Android](https://github.com/woocommerce/woocommerce-android/issues) repositories.
```

File: .github/ISSUE_TEMPLATE/rest-api-mobile.yml (hunk 1 above)
Comment (malinajirka) [reply .github/ISSUE_TEMPLATE/rest-api-mobile.yml]: Totally missed this one, great catch, thanks!

**Comment by malinajirka** [comment]
Thanks for the review @coreymckrill ! I've committed all the suggested changes 🙇.

**Review by coreymckrill — APPROVED** [review]
👍

**Pull request merged**